src/dvclive/lightning.py

```diff
@@ -62,6 +62,7 @@ def __init__(  # noqa: PLR0913
         save_dvc_exp: bool = False,
         dvcyaml: bool = True,
         cache_images: bool = False,
+        exp_name: Optional[str] = None,
```

Comment:
Will add `exp_message` and update other args for this callback in a separate PR.